.Rbuildignore

```diff
@@ -28,3 +28,4 @@
 ^vignettes/[^-]+.gif$
 ^CRAN-SUBMISSION$
 ^CODE_OF_CONDUCT\.md$
+^paper.*$
```

Comment:
Nit: just `^paper$`? that will exclude the whole directory. At a glance I thought we were adding new top-level files instead of a new directory.